content/docs/0.11.x/reference/bridge/oauth/index.md

```diff
@@ -50,80 +67,72 @@ Response :
 ```
 Request :
 
-   POST <OAuth_Service>/token_decision
+   POST <token_endpoint>
 
    {
      "state" : <state>,
-     "code" : <authorization_code>
+     "code" : <authorization_code>,
+     "scope": "openid",
+     "client_id": <client_id>,
+     "client_secret": <client_secret>, 
+     "nonce": <nonce>,
+     "code_verifier": <code_verifier>
    }
 
 Response :
 
   {
-   "user" : <USER_IDENTIFIER>
+   "id_token": <id_token>,
+   "access_token": <access_token>,
+   "refresh_token": <refresh_token>,
+   "expires_in": <expires_in>
   }
 ```
 
-OAuth service must implement logic to generate authorization requests, obtain tokens for valid authorization codes and validate permission checks based on token contents or user information. Furthermore, authorization request and token request should be linked ideally through state. OAuth-service is expected to follow recommended best practices such as use of [PKCE](https://tools.ietf.org/html/rfc7636) and JWK validations where applicable.
+The identity provider must implement logic to generate authorization requests, obtain tokens for valid authorization codes and validate permission checks based on token contents or user information. Furthermore, authorization request and token request should be linked ideally through state, nonce, code challenge and code verifier. The identity provider is expected to follow recommended best practices such as use of [PKCE](https://tools.ietf.org/html/rfc7636) and JWK validations where applicable.
```

Comment:
Is `nonce` a typo?

Reply:
No https://openid.net/specs/openid-connect-core-1_0.html#:~:text=auth_time%20response%20parameter.)-,nonce,-String%20value%20used